doc/how_to/components/construct_panes.md

```diff
@@ -0,0 +1,30 @@
+# Construct Panes
+
+There are two main ways to construct a pane - explicitly or automatically.
+
+To explicitly construct a pane, use one of the pane types listed in the [reference gallery](../../reference/index.html#panes). For example, you can create a Markdown pane as follows:
+
+```{pyodide}
+import panel as pn
+pn.extension()
+
+pn.pane.Markdown('''
+# H1
+## H2
+### H3
+''')
+```
+
+Alternatively, you can create a pane using the `pn.panel()` utility to automatically infer the pane type from the object being passed as the argument:
+
+```{pyodide}
+png = pn.panel('https://upload.wikimedia.org/wikipedia/commons/4/47/PNG_transparency_demonstration_1.png', width=500)
+
+png
+```
+
+:::{admonition} See Also
+:class: seealso
```

Comment:
Is this a valid class?

Reply:
[According to MyST](https://myst.tools/docs/spec/admonitions#:~:text=hint%22%20%7C%20%22important%22%20%7C%20%22note%22%20%7C%20%22-,seealso,-%22%20%7C%20%22tip%22%20%7C%20%22warning%22)
), yes. Although it appears that you are flattening the styling of anything but {important}, is that correct? Sorry, I'm not fully understanding this system yet.